lib/compat/wordpress-6.2/block-editor-settings.php

```diff
@@ -16,15 +16,12 @@ function gutenberg_get_block_editor_settings_6_2( $settings ) {
 	if ( wp_theme_has_theme_json() ) {
 		// Add the custom CSS as separate style sheet so any invalid CSS entered by users does not break other global styles.
 		$settings['styles'][] = array(
-			'css'            => gutenberg_get_global_styles_custom_css(),
+			'css'            => wp_get_global_styles_custom_css(),
```

Comment:
Given that we have a `gutenberg_get_block_editor_settings` in `lib/block-editor-settings.php`, why do we need `lib/compat/wordpress-6.2/block-editor-settings.php` as well?

Reply:
Good catch. I'll remove that.